LiteCore/Support/Logging.cc

```diff
@@ -230,25 +234,20 @@ namespace litecore {
         va_end(args);
     }
 
-
     // The default logging callback writes to stderr, or on Android to __android_log_write.
     static void defaultCallback(const LogDomain &domain, LogLevel level,
-                                const char *fmt, va_list args)
-    {
-        #ifdef __ANDROID__
-            static const int kLevels[5] = {ANDROID_LOG_DEBUG, ANDROID_LOG_INFO, ANDROID_LOG_INFO,
-                                           ANDROID_LOG_WARN, ANDROID_LOG_ERROR};
-            static char source[100] = "LiteCore";
-            auto name = domain.name();
-            if (name[0]) {
-                strcat(source, " ");
-                strcat(source, name);
-            }
-            vsnprintf(sFormatBuffer, sizeof(sFormatBuffer), fmt, args);
-            __android_log_write(kLevels[(int)level], source, sFormatBuffer);
+                                    const char *fmt, va_list args){
+
+        auto name = domain.name();
+        static const char *kLevels[] = {"***", "", "", "WARNING", "ERROR"};
+        #if ANDROID
+            stringstream buf;
+            LogDecoder::writeHeader(kLevels[(int) level], name, buf);
+            static const int androidLevels[5] = {ANDROID_LOG_DEBUG, ANDROID_LOG_INFO,
+                                                 ANDROID_LOG_INFO, ANDROID_LOG_WARN,
+                                                 ANDROID_LOG_ERROR};
+            __android_log_vprint(androidLevels[(int) level], buf.str().c_str(), fmt, args);
```

Comment:
The result of `LogDecoder::writeHeader` is a combination of the domain name and level with brackets, like `[Query] WARNING:`. That doesn't seem like the right thing to pass as the tag to the log — isn't it supposed to be just `LiteCore` plus the domain name, as before?